src/AWS.Deploy.ServerMode.Client/AWS.Deploy.ServerMode.Client.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
This is used by the toolkit which can only use .NET Standard 2.0. We need to not move this to `netstandard2.1`. You might want to add a comment here why we have to keep this at .NET Standard 2.0 to prevent somebody from attempting to change this in the future.

Reply:
Done